src/app/delegation_backend/src/constants.go

```diff
@@ -11,16 +14,37 @@ const DELEGATION_WHITELIST_LIST = "Form Responses 1"
 const DELEGATION_WHITELIST_COLUMN = "E"
 
 // Production
-const DELEGATION_WHITELIST_SPREADSHEET_ID = "1xiKppb0BFUo8IKM2itIx2EWIQbBzUlFxgtZlKdnrLCU"
-const CLOUD_BUCKET_NAME = "foundation-delegation-snark-work"
+const PROD_WHITELIST_SPREADSHEET_ID = "1xiKppb0BFUo8IKM2itIx2EWIQbBzUlFxgtZlKdnrLCU"
+const PROD_CLOUD_BUCKET_NAME = "foundation-delegation-uptime"
+
+const TEST_WHITELIST_SPREADSHEET_ID = "1NODwwcVxLNnCI4XnIrGdGBSjointN4MZ8QZ7wqgtSTQ"
+const TEST_CLOUD_BUCKET_NAME = "georgeee-delegation-test-1"
+
+func CloudBucketName() string {
+	if os.Getenv("TEST") == "" {
+		return PROD_CLOUD_BUCKET_NAME
+	} else {
+		return TEST_CLOUD_BUCKET_NAME
+	}
+}
+
+func WhitelistSpreadsheetId() string {
+	if os.Getenv("TEST") == "" {
+		return PROD_WHITELIST_SPREADSHEET_ID
+	} else {
+		return TEST_WHITELIST_SPREADSHEET_ID
+	}
+}
 
 var PK_PREFIX = [...]byte{1, 1}
 var SIG_PREFIX = [...]byte{1}
+var BLOCK_HASH_PREFIX = [...]byte{1}
 
 const NETWORK_ID = 1  // mainnet
 const PK_LENGTH = 33  // one field element (32B) + 1 bit (encoded as full byte)
 const SIG_LENGTH = 64 // one field element (32B) and one scalar (32B)
 
+// we use state hash code here, although it's not state hash
```

Comment:
It's complicated: it would require another round of data migration and some more action on Ontab's side. We might do it if we'll be doing such a pass for other reasons in future though.